### DISCUSSION TITLE
Process/Upscale to JPG?

### DISCUSSION THREAD

**Discussion by brianjz** [vladmandic/sdnext#1568] [General]
I looked around the settings, but only found an "all" option. Is there a way to save just the images from the "Process Image" upscale tab to JPG instead of PNG automatically? Those PNGs are sometimes in the 6-7MB range and I know JPG would be smaller. If not, I have workarounds. Thanks.

**Comment by vladmandic** [comment]
no, its all or nothing.

**Comment by Tillerz** [comment]
I do not save anything automatically and just use "save" button on what I actually want to keep. Saves a lot of space.

**Reply by brianjz** [reply]
I may look into that. Space isn't really an issue, but I do collect a lot of garbage sometimes when trying to find that one render.
